### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by VirginiaBalseiro**
- Added instructions on how to run the project

### CODE REVIEW

**Comment by VirginiaBalseiro** [comment]
> Line 39 should read Run `node server.js'

It works with `npm start`, we have a script.

File: README.md [hunk 1]
```diff
@@ -27,3 +27,25 @@ We will be building this project with JavaScript: React on the frontend and Node
...
+
+### Backend
+
+- See [here](https://github.com/Junior-Devs/2020-Junior-Devs-Project/wiki/Local-Database-Setup-for-Testing-Backend) how to set up the local database.
+- In the main directory, run `npm install`
+- Run `npm start`. If all went well you should see "NodeJS Server Running"
+
+### Frontend
+- `cd frontend`
+- Run `npm install` (the frontend and backend are two separate projects with separate `package.json` file, so you need to do this again inside the `frontend` directory
```
Comment (Enzo-Parodi) [inline README.md:43]: *files* instead of *file*.

File: README.md [hunk 2]
```diff
@@ -27,3 +27,25 @@ We will be building this project with JavaScript: React on the frontend and Node
...
+- Run `npm run dev`. If all went well, you should see "Server running at http://localhost:1234". You can now open the project at http://localhost:1234.
+
+*** Important ***: There is a `HelloWorld.jsx` sample component with its corresponding test file (`HelloWorld.spec.jsx`) and story (`HelloWorld.stories.jsx`). DO NOT delete these, they are there as templated to help you and others get started.
```
Comment (Enzo-Parodi) [inline README.md:46]: *templates* instead of *templated*

File: README.md [hunk 3]
```diff
@@ -27,3 +27,25 @@ We will be building this project with JavaScript: React on the frontend and Node
...
+### Frontend
+- `cd frontend`
+- Run `npm install` (the frontend and backend are two separate projects with separate `package.json` file, so you need to do this again inside the `frontend` directory
+- Run `npm run dev`. If all went well, you should see "Server running at http://localhost:1234". You can now open the project at http://localhost:1234.
+
+*** Important ***: There is a `HelloWorld.jsx` sample component with its corresponding test file (`HelloWorld.spec.jsx`) and story (`HelloWorld.stories.jsx`). DO NOT delete these, they are there as templated to help you and others get started. 
+
+### To run Storybook:
+
+- `cd frontend` if you're not already the `frontend` directory
```
Comment (Enzo-Parodi) [inline README.md:50]: ...not already **in** the `frontend` directory-

**Pull request merged**